### PR TITLE
Solution for Obstacle #72946 and package/dependency updates

### PR DESCRIPTION
**Pull request by ChrisLearner62**
Solution for Obstacle #72946 and package/dependency updates for below files:
- package-lock.json
- package.json

### CODE REVIEW

File: cypress/e2e/chris.cy.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+describe("", () => {
+    it("Obstacle #72946", () => {
+        cy.visit('https://obstaclecourse.tricentis.com/Obstacles/72946');
+        // download the xml file
+        cy.request({
+            url: '/Obstacles/GetObstacleFile?obstacleId=72946&filename=Catalog.xml',
+            method: 'GET',
+            encoding: 'binary',
+          }).then((response) => {
+            const fileName = 'Catalog.xml';
```
Comment (jun1983) [inline cypress/e2e/chris.cy.ts:12]: Do you need to write response into a local file then read it again?
I feel you can do it directly

File: cypress/e2e/chris.cy.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            cy.writeFile(fileName, response.body, 'binary');
+          });
+        // convert xml to json
```
Comment (jun1983) [inline cypress/e2e/chris.cy.ts:15]: Why not use xpath to retrieve the node value?

File: package.json [hunk 3]
```diff
@@ -14,7 +14,10 @@
     "cypress-multi-reporters": "^1.6.3",
     "mocha": "^10.2.0",
     "mocha-junit-reporter": "^1.23.3",
-    "mochawesome": "^7.1.3",
-    "typescript": "^5.0.2"
+    "mochawesome": "^7.1.3"
+  },
+  "devDependencies": {
```
[the remark this answers is not in the document]
Comment (ChrisLearner62) [reply package.json]: fixed

File: cypress/e2e/chris.cy.ts (hunk 1 above)
Comment (ChrisLearner62) [reply cypress/e2e/chris.cy.ts]: Update code without reading and writing files.

File: cypress/e2e/chris.cy.ts (hunk 2 above)
Comment (ChrisLearner62) [reply cypress/e2e/chris.cy.ts]: not applicable any more

**Pull request closed without merging**